src/role-helpers.js

```diff
@@ -191,7 +194,20 @@ function prettyRoles(dom, {hidden}) {
             computedStyleSupportsPseudoElements:
               getConfig().computedStyleSupportsPseudoElements,
           })}":\n`
+
           const domString = prettyDOM(el.cloneNode(false))
+
+          if (includeDescription) {
```

Comment:
@eps1lon not related to this PR, but should we also do a `includeName`?
It seems like we're already providing the property [here](https://github.com/testing-library/dom-testing-library/blob/main/src/queries/role.js#L229) but we're not doing anything with it.

Reply:
I think we originally decide to always include it to help people discover the `name` option (would have to check the original PR though).

Though I don't think this necessarily applies to `description` since it's
1. more likely to not be set
2. quite noisy (concerning length) if you don't care about it

Reply:
I'll take a look at the history.
If we indeed decided to always include it (which makes sense imho), I'll remove the code where we now add the `includeName` argument.